### PR TITLE
ClimaCore Makie Plots

### LINKED ISSUE #222: Add ability to plot 2D Fields on the sphere
- [x]  Unwrapped panels of the cubed sphere in 2D with Plots.jl
- [ ]  3D visualization with Makie

### PR DESCRIPTION
**Pull request by jakebolewski**
Setup the infrastructure for 3D / 2D Makie plotting recipes for ClimaCore object types. Add example rectangle and cubed sphere field plots. 

Closes #222

### CODE REVIEW

File: lib/ClimaCoreMakie/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2021 Climate Modeling Alliance
```
Comment (simonbyrne) [inline lib/ClimaCoreMakie/LICENSE.md:1]: This should be a `NOTICE` file, and the full apache license should be in `LICENSE` (see ClimaCore or ClimaCoreVTK)

Comment (jakebolewski) [reply lib/ClimaCoreMakie/LICENSE.md]: should be fixed now

**Comment by jakebolewski** [comment]
This is not at all complete but mostly to get something started, need to incorporate sphere level slicing / volume box rendering for 3D plots as a followup.

**Pull request merged**